math/src/polismath/math/conversation.clj

```diff
@@ -18,13 +18,8 @@
     [plumbing.core :as plmb]
     [plumbing.graph :as graph]
     [bigml.sampling.simple :as sampling]
-    ;[alex-and-georges.debug-repl :as dbr]
     [taoensso.timbre :as log]
-    [clojure.spec.gen.alpha :as gen]
-    [clojure.test.check.generators :as generators]
```

Comment:
I think the generators are actually ns must actually be required for some of this code to work (such as `s/with-gen` below), but not 100% certain. In any case, may still be useful to keep around as it could be helpful for testing/dev.

Reply:
Included again but commented out as were flagged by Snyk